src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -31,26 +31,17 @@
     type: command
     short-summary: Upload a file to a storage blob.
     long-summary: Creates a new blob from a file path, or updates the content of an existing blob with automatic chunking and progress notifications.
-    examples:
-        - name: Upload to a blob.
-          text: az storage blob upload -f /path/to/file -c MyContainer -n MyBlob
 """
 
 helps['storage file upload'] = """
     type: command
     short-summary: Upload a file to a share that uses the SMB 3.0 protocol.
     long-summary: Creates or updates an Azure file from a source path with automatic chunking and progress notifications.
-    examples:
-        - name: Upload to a local file to a share.
-          text: az storage file upload -s MyShare -source /path/to/file
```

Comment:
Would keep.

Reply:
Same as with `az storage blob upload`. The existing example is not very useful but improved examples would be helpful. Suggestions welcome.